02LessonTwo/in-class-project/index.html

```diff
@@ -5,6 +5,32 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <center><img src="https://s-media-cache-ak0.pinimg.com/736x/80/a1/c5/80a1c5ce85954a56487e33b5c8e8b466.jpg" alt="Muffin Market"/></center>
+  <h1 align="center">Muffin Market</h1>
+  <h2 align="center"><em>Fluffy muffins for all!</em></h2>
+  <br>
+  <br>
+  <br>
+  <table border="1" align="center">
+    <tr>
+      <th colspan="3"><h3>Muffins for sale!</h3></th>
+    </tr>
+    <tr>
+      <td><img src="http://a66c7b.medialib.glogster.com/dianahan/media/cd/cd1fe442dd38ab207e86bfb0c7e92894424fa4cb/muffin2.png" alt="Blueberry muffin - $10"/></th>
+      <td><img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSRBIIrd4GmRkZ7DH2vtoyyVoy8sx_525dI928TyYlxkm7uiNmK" alt="Bran muffin - $8"/></th>
+      <td><img src="https://s-media-cache-ak0.pinimg.com/564x/0e/e3/6f/0ee36f74077f3ae6e829e1cf103d6efc.jpg" alt="Banana Nut muffin - $8"/></th>
+    </tr>
+    <tr>
+      <td><p align="center">Blueberry - $10</p></td>
+      <td><p align="center">Bran - $8</p></td>
+      <td><p align="center">Banana Nut - $8</p></td>
+    </tr>
+  </table>
+  <br>
+  <br>
+  <br>
+  <center><a href="http://wwww.google.com">Muffin Market</a></center>
+  <p align="center">Phone: 555-555-5555</p>
+  <center><a href="mailto:webmaster@example.com">Email us!</a></center>
 </body>
 </html>
```

Comment:
Love me a fluffy muffin. Great work here. You show a great grasp of the lesson, and a sharp eye for design and layout.